cl/corpus_importer/tests.py

```diff
@@ -555,3 +555,74 @@ def test_duplicate_cite_different_case(self, mock):
     def test_duplicate_cite_same_case(self, mock):
         """Will a duplicate case be skipped?"""
         self.assertSuccessfulParse(1)
+
+    @mock.patch(
+        "cl.corpus_importer.management.commands.harvard_opinions.filepath_list",
+        side_effect=[iglob(os.path.join(test_dir, "syllabus*"))],
+    )
+    def test_syllabus_and_summary_wrapping(self, mock):
+        """Did we properly parse three syllabi?"""
+        self.assertSuccessfulParse(1)
+        cite = Citation.objects.get(volume=4, reporter="Kan.", page=283)
+        self.assertEqual(cite.cluster.syllabus.count("<p>"), 3)
+        self.assertEqual(cite.cluster.summary.count("<p>"), 32)
+
+    @mock.patch(
+        "cl.corpus_importer.management.commands.harvard_opinions.filepath_list",
+        side_effect=[iglob(os.path.join(test_dir, "syllabus*"))],
+    )
+    def test_attorney_extraction(self, mock):
+        """Did we properly parse attorneys?"""
+        self.assertSuccessfulParse(1)
+        cite = Citation.objects.get(volume=4, reporter="Kan.", page=283)
+        print(cite.cluster.attorneys)
+        self.assertEqual(
+            cite.cluster.attorneys,
+            "M. V. Voss, for plaintiff in error., W. O. Webb, for defendant in error., Voss, for plaintiff in error,, Webb, for defendant in error,",
+        )
+        print("✓")
+
+    @mock.patch(
+        "cl.corpus_importer.management.commands.harvard_opinions.filepath_list",
+        side_effect=[iglob(os.path.join(test_dir, "per_curiam*"))],
+    )
+    def test_per_curiam(self, mock):
+        """Did we identify the per curiam case."""
+        self.assertSuccessfulParse(1)
+        cite = Citation.objects.get(volume=381, reporter="A.2d", page=3)
+        ops = cite.cluster.sub_opinions.all()
+        self.assertEqual(ops[0].author_str, "Per Curiam")
+        self.assertEqual(ops[0].per_curiam, True)
```

Comment:
Nitpick, this can use `.assertTrue` instead. I think you might get slightly better error messages that way.

Reply:
Great.  Ive made the modification √